c2corg_ui/static/js/utils.js

```diff
@@ -0,0 +1,58 @@
+goog.provide('app.utils');
+
+goog.require('goog.asserts');
+
+
+/**
+ * @param {string} module The document type.
+ * @param {string} documentId The document id.
+ * @param {string} culture Culture.
+ * @return {string} Url.
+ */
+app.utils.buildDocumentUrl = function(module, documentId, culture) {
+  return '/{module}/{id}/{culture}'
+    .replace('{module}', module)
+    .replace('{id}', documentId)
+    .replace('{culture}', culture);
+};
+
+
+/**
+ * Cultures ordered by their relevance.
+ * @const
+ * @type {Array.<string>}
+ */
+app.utils.LANG_PRIORITY = ['fr', 'en', 'it', 'de', 'es', 'ca', 'eu'];
```

Comment:
Because we have to figure out what culture of the document should be shown when several cultures are available.
See comments in https://github.com/c2corg/v6_api/issues/83